src/components/Calculator.js

```diff
@@ -1,32 +1,26 @@
-import React from 'react';
-import Buttons from './Buttons';
+import React, { useState } from 'react';
 import Output from './Output';
+import Buttons from './Buttons';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: '0',
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
+  const [calculation, setCalculation] = useState({
+    total: '0',
+    next: null,
+    operation: null,
+  });
 
-  handleClick = (event) => {
-    const values = calculate(this.state, event.target.innerText);
-    this.setState(values);
+  const handleClick = (event) => {
+    const values = calculate(calculation, event.target.innerText);
+    setCalculation(values);
   };
 
-  render() {
-    return (
-      <div className="calculator">
-        <Output state={this.state} />
-        <Buttons handleClick={this.handleClick} />
-      </div>
-    );
-  }
-}
+  return (
+    <div className="calculator">
+      <Output calculation={calculation} />
+      <Buttons handleClick={handleClick} />
+    </div>
```

Comment:
- [ ] [optional] It would be nice adding more width and height to your calculator currently its too small, 
![image](https://user-images.githubusercontent.com/61408860/206796569-79be9b25-cb05-4164-a643-a822fa11e3ff.png)